cmd-import.c

```diff
@@ -263,8 +263,10 @@ static int csv_parse_accounts(FILE *fp, struct list_head *account_list,
 	if (url_index == -1 && username_index == -1 &&
 	    password_index == -1 && extra_index == -1 &&
 	    name_index == -1 && grouping_index == -1 &&
-	    fav_index == -1)
+	    fav_index == -1) {
+		printf("Could not read the CSV header at the first line\n");
```

Comment:
Could you use the `die(const char *err, ...)` function from util.c?

Reply:
I changed this to die and also added the signed off commit message.